onadata/apps/logger/models/xform.py

```diff
@@ -818,6 +818,13 @@ def _set_title(self):
                 _("Title shouldn't have any invalid xml "
                   "characters ('>' '&' '<')"))
 
+        pattern = str("http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|"
```

Comment:
Should we update the pattern to also match URLs that don't include the HTTP/HTTPS protocols?

![Screenshot 2021-02-01 at 12 10 31 PM](https://user-images.githubusercontent.com/25849009/106437714-98fc5100-6486-11eb-8f2e-b0b2cea52d5d.png)

Reply:
Yes we should. Let me update